src/protocol.h

```diff
@@ -55,220 +55,255 @@ class CMessageHeader
 
 /**
  * Bitcoin protocol message types. When adding new message types, don't forget
- * to update allNetMessageTypes in protocol.cpp.
+ * to update g_all_net_message_types below.
  */
 namespace NetMsgType {
-
 /**
  * The version message provides information about the transmitting node to the
  * receiving node at the beginning of a connection.
  */
-extern const char* VERSION;
+inline constexpr const char* VERSION{"version"};
```

Comment:
You can drop `inline` as `constexpr` is implicitly `inline`.
> A static data member (i.e. static class member or namespace-scope variable) declared constexpr on its first declaration is implicitly an inline variable.
https://en.cppreference.com/w/cpp/language/inline